java/src/main/java/ai/onnxruntime/OrtSession.java

```diff
@@ -790,28 +790,6 @@ public void addDnnl(boolean useArena) throws OrtException {
       addDnnl(OnnxRuntime.ortApiHandle, nativeHandle, useArena ? 1 : 0);
     }
 
-    /**
-     * Adds OpenVINO as an execution backend.
-     *
-     * @param deviceId The id of the OpenVINO execution device.
-     * @throws OrtException If there was an error in native code.
-     */
-    public void addOpenVINO(String deviceId) throws OrtException {
```

Comment:
I realise these need shared library support which isn't currently possible in Java, but is there some other reason they are being removed? Are these two providers deprecated?

Reply:
What do you mean isn't currently possible ?

Reply:
The changes which broke out OpenVINO, DNNL and TensorRT into separate shared libraries broke the Java API wrappers around those providers. I opened #6553 as we'll package those shared libraries into the jar, and so need to be able to load them from a non-standard place, but as there's been no progress there they are broken in the current release and it's been broken since those providers are converted into shared libraries. When this (https://communities.sas.com/t5/SAS-Communities-Library/SAS-and-Microsoft-collaborate-to-democratize-the-use-of-Deep/ta-p/730072) lands it'll break CUDA too.

Reply:
BTW I've got the code ready to go to move these providers over to the new API point (with the struct), but I can't test it because Java can't load the library.